### PR TITLE
Receive Incoming Emails

### PR DESCRIPTION
**Pull request by ccrawford13**
Receive incoming emails with Mailgun

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,7 @@
 
 # Ignore application configuration
 /config/application.yml
+
+config/application.yml
```
Comment (felixclack) [inline .gitignore:22]: These lines are duplicates of line 20.

**Comment by felixclack** [comment]
This looks good. Did you test it on Heroku?

**Comment by ccrawford13** [comment]
Yes - Took some additional commits and configuration files to get everything working in production, but I was able to get the sent email to post to the production app.

**Comment by felixclack** [comment]
Great. 

—
Thanks,
Felix

**Pull request merged**